dev/io.openliberty.accesslists.internal/bnd.bnd

```diff
@@ -0,0 +1,47 @@
+#*******************************************************************************
+# Copyright (c) 2021 IBM Corporation and others.
+# All rights reserved. This program and the accompanying materials
+# are made available under the terms of the Eclipse Public License v1.0
+# which accompanies this distribution, and is available at
+# http://www.eclipse.org/legal/epl-v10.html
+#
+# Contributors:
+#     IBM Corporation - initial API and implementation
+#*******************************************************************************
+-include= ~../cnf/resources/bnd/bundle.props
+bVersion=1.0
+
+Bundle-Name: Access Lists
+Bundle-SymbolicName: io.openliberty.accesslists.internal; singleton:=true
+Bundle-Description: Provides access list and filter function for TCP, UDP \
+and HTTP channels version ${bVersion}
+
+WS-TraceGroup: ChannelFramework
+
+# For each exported package, create (in that package) a package-info.java
+# file, and place an @version javadoc tag in package-level javadoc. 
+# Append ";provide:=true" if this bundle also provides an implementation
+# for the exported API.
+Export-Package: \
+	io.openliberty.accesslists;version="1.0";provide:=true, \
```

Comment:
is there a reason for exporting the version here rather than in a `package-info.java`?

Reply:
No, just a copy/paste error. I will fix.

Reply:
done.